recipes/exiv2/all/conanfile.py

```diff
@@ -0,0 +1,160 @@
+import os
+import textwrap
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class Exiv2Conan(ConanFile):
+    name = "exiv2"
+    description = "Exiv2 is a C++ library and a command-line utility " \
+                  "to read, write, delete and modify Exif, IPTC, XMP and ICC image metadata."
+    license = "GPL-2.0"
+    topics = ("conan", "image", "exif", "xmp")
+    homepage = "https://www.exiv2.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_png": [True, False],
+        "with_xmp": [False, "bundled", "external"],
+        "with_curl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_png": True,
+        "with_xmp": "bundled",
+        "with_curl": False,
+    }
```

Comment:
So initialize it here as a list (it is not documented how `ConanFile` initializes it. It might have been a tuple)
```suggestion
    }
    provides = []
```